src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1340,10 +1340,21 @@ protected void deleteFile(File deletedFile, File dir, File targetDir, String tar
         File targetFile = getTargetFile(deletedFile, dir, targetDir, targetFileName);
         debug("Target file exists: " + targetFile.exists());
         if (targetFile.exists()) {
-            if (targetFile.delete()){
-                info("Deleted file" + targetFile.getCanonicalPath());
+            if (targetFile.isDirectory()) {
+                try {
+                    FileUtils.deleteDirectory(targetFile);
+                } catch (IllegalArgumentException e) {
+                    debug("Could not delete directory " + e.getMessage());
```

Comment:
Use sentences for the new debug and error messages (lines 1347, 1349, 1458, 1460), e.g. they are missing a period.